doc/api/child_process.md

```diff
@@ -1325,6 +1325,25 @@ then this will be `null`.
 `subprocess.stderr` is an alias for `subprocess.stdio[2]`. Both properties will
 refer to the same value.
 
+```js
+const { spawn } = require('child_process');
+
+const filePath = process.argv[1];
+
+const subprocess = spawn('cat', [filePath], {
+  stdio: ['inherit', 'inherit', 'pipe']
+});
+
+console.log('stderr some data');
+console.log('server:', subprocess.stderr.server);
+console.log('handle:', subprocess.stderr._handle);
```

Comment:
The underscore prefixed properties should not be included. Those are considered to be private APIS

Reply:
I'm working on it.